cluster/saltbase/salt/top.sls

```diff
@@ -40,6 +40,9 @@ base:
     - kube-addons
 {% if grains['cloud'] is defined and grains['cloud'] == 'azure' %}
     - openvpn
+{% else %}
+    - docker
+    - sdn
```

Comment:
Ah sorry, it did not enable sdn on everything as that is conditional, but it did put docker on everything.  Apologies, missed that.  @rajatchopra - can we update to not require docker on the kubernetes-master, but just have the sdn bit?